src/rt/sections_elf_shared.d

```diff
@@ -927,13 +927,12 @@ struct tls_index
 }
 
 version(LDC)
+
+version (linux) version (PPC) version = linux_PPC_Any;
+version (linux) version (PPC64) version = linux_PPC_Any;
+
```

Comment:
whoops :) This addition should be within the `{` parens of `version(LDC)`

Reply:
Done